graph/adjacency/ThingAdjacency.java

```diff
@@ -164,12 +193,8 @@
 
         void deleteAll();
 
-        ThingEdge cache(ThingEdge edge);
-
```

Comment:
we no longer cache persisted ThingEdge instances, since this could blow up memory (imagine traversing a read-only supernode, all edges will be cached in memory until we are finished with that read vertex. 

Alternative source of memory blowup could be a modified vertex that is cached in the ThingGraph. The vertex caches all buffered edges (it must) - however it doesn't have to also cache all unmodified edges.